configure.ac

```diff
@@ -1316,7 +1316,7 @@ AC_CHECK_FUNCS(qb_ipcc_connect_async,
 
 dnl libqb 2.0.2+ (2020-10)
 AC_CHECK_FUNCS(qb_ipcc_auth_get,
-               AC_DEFINE(HAVE_IPCC_AUTH_GET, 1,
+               AC_DEFINE(HAVE_QB_IPCC_AUTH_GET, 1,
```

Comment:
Actually (thankfully) this shouldn't be necessary. AC_CHECK_FUNCS() will already define that, so we were just unnecessarily defining the alternate name. We can just drop the second argument altogether.

Reply:
Then let's make it consistent if we are sure it works the same on all platforms/versions we do support.